web/src/pages/api/post/create.ts

```diff
@@ -25,7 +26,14 @@ export default async function create(req: Request, res: NextApiResponse) {
   if (!prompt.trim()) {
     return res.status(400).json({ error: "Empty prompt provided" });
   }
-
+  
+  // Check if the prompt is profane
+  //added these two words to the profanity list
+  profanity.addWords(['sexy', 'nude']);
```

Comment:
is hot, and bikini also in there? those were the words used to generate the nsfw images too